docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:21-slim
```

Comment:
let's use 22 latest

Reply:
and please pin to specific version

Reply:
Valentine adviced against this, since 21 is the only version that have been verified to have no leaks, he said 20 and 22 are less favourable.

Reply:
gotcha, thanks

Reply:
Actually 20 is the only one has possible leaks other than that both 21 and 22 are good.